packages/blocks/src/__internal__/content-parser/file-exporter/exporter-style.ts

```diff
@@ -121,9 +121,118 @@ export const globalCSS = css`
     --affine-palette-shape-black: #000;
     --affine-palette-shape-grey: #c2c2c2;
     --affine-tooltip: #424149;
+    --affine-shadow-1: 0px 0px 4px 0px rgba(66, 65, 73, 0.14);
```

Comment:
This naming looks a bit off 🤔

Reply:
The properties and values here refer to this code, https://github.com/toeverything/design/blob/main/packages/theme/src/index.ts
<img width="706" alt="image" src="https://github.com/toeverything/blocksuite/assets/53252747/c71e67d0-17d9-4ca2-9b6a-66b6c297987e">

Reply:
I haven't come up with a final method for css variables in exporting file.

The first way is to rewrite css variables like the current way.

The second way is to import theme package in blocksuite.

The third way is to collect all css variables when exporting file. Like following,
```typescript
  const cssAffineVars = {};
  const styles = window.getComputedStyle(document.documentElement);

  for (const prop of styles) {
    if (prop.startsWith("--affine-")) {
      cssAffineVars[prop] = styles.getPropertyValue(prop);
    }
  }
```